### ISSUE TITLE
Can I subscribe to a collection at a time

### ISSUE THREAD

**Issue by zhouZ813** [emqtt/CocoaMQTT#244]
If I have a lot of topics and a lot of people. Is it a bit performance intensive to iterate through subscriptions?

**Comment by HJianBo** [comment]
CocoaMQTT does not support that now, I'll add a new function later.

Ex:
```swift
func subscribe(_ topics: [(String, CocoaMQTTQOS)]) -> UInt16
```

**Comment by zhouZ813** [comment]
thank you！

**Comment by HJianBo** [comment]
Hi, @zhouZ813 The feature was released in the v1.2.0 yesterday.

Thanks for your advice!
